torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4342,6 +4373,21 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            op=torch.fmin,
            dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
            sample_inputs_func=sample_inputs_max_min_binary,),
+    OpInfo('fmod',
+           dtypes=all_types_and(torch.float16, torch.bool),
+           sample_inputs_func=sample_inputs_fmod_remainder,
+           skips=(
+               SkipInfo('TestCommon', 'test_variant_consistency_jit'),
+               SkipInfo('TestCommon', 'test_variant_consistency_eager'),
```

Comment:
`test_variant_consistency_eager_fmod_cuda_float32` and `test_variant_consistency_eager_fmod_cpu_float32` were failing. Have skipped these tests for now, any help on this will be appreciated.

```
FAIL: test_variant_consistency_eager_fmod_cuda_float32 (__main__.TestCommonCUDA)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_utils.py", line 1033, in wrapper
    method(*args, **kwargs)
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_utils.py", line 1033, in wrapper
    method(*args, **kwargs)
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_device_type.py", line 292, in instantiated_test
    result = test_fn(self, *args)
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_device_type.py", line 266, in test_wrapper
    return test(*args, **kwargs)
  File "/home/krshrimali/git/krshrimali/pytorch/test/test_ops.py", line 326, in test_variant_consistency_eager
    _test_consistency_helper(samples, variants)
  File "/home/krshrimali/git/krshrimali/pytorch/test/test_ops.py", line 310, in _test_consistency_helper
    variant_forward = variant(cloned,
AssertionError: RuntimeError not raised
```

Reply:
Ah right. Looks like `fmod` resizes the self on inplace variant (which is incorrect behavior). Can you add comment for the same and file an issue
cc: @mruberry

Reply:
Sure, @kshitij12345! I'll add a comment in the `skips` arg, and link the issue.